client/jetpack-onboarding/steps/stats.jsx

```diff
@@ -0,0 +1,132 @@
+import Tile from 'components/tile-grid/tile';
+import TileGrid from 'components/tile-grid';
+import { addQueryArgs } from 'lib/route';
+import { getUnconnectedSiteUrl } from 'state/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+import { JETPACK_ONBOARDING_STEPS as STEPS } from '../constants';
+import { saveJetpackOnboardingSettings } from 'state/jetpack-onboarding/actions';
+
+class JetpackOnboardingStatsStep extends React.Component {
+	componentDidUpdate() {
+		const { action, activatedStats, isConnected } = this.props;
+
+		if ( isConnected && activatedStats === false && action === 'activate_stats' ) {
+			this.activateStats();
+		}
+	}
+
+	handleActivateStats = () => {
+		this.props.recordJpoEvent( 'calypso_jpo_stats_clicked' );
+
+		if ( ! this.props.isConnected ) {
+			return;
+		}
+
+		this.activateStats();
+	};
+
+	handleStatsNextButton = () => {
+		this.props.recordJpoEvent( 'calypso_jpo_stats_next_clicked' );
+	};
+
+	activateStats() {
+		this.props.saveJetpackOnboardingSettings( this.props.siteId, {
+			stats: true,
+		} );
+	}
+
+	renderSuccess() {
+		const { getForwardUrl, translate } = this.props;
+		const headerText = translate( 'Success! Jetpack is now collecting valuable stats.' );
+
+		return (
+			<Fragment>
+				<FormattedHeader headerText={ headerText } />
+
+				<TileGrid>
+					<Tile
+						buttonLabel={ translate( 'Continue' ) }
+						image={ '/calypso/images/illustrations/type-business.svg' }
+						onClick={ this.handleStatsNextButton }
+						href={ getForwardUrl() }
+					/>
+				</TileGrid>
+			</Fragment>
+		);
+	}
+
+	renderActionTile() {
+		const { isConnected, siteUrl, translate } = this.props;
+		const headerText = translate( 'Keep track of your visitors with Jetpack.' );
+		const subHeaderText = translate(
+			'Keep an eye on your success with simple, concise, and mobile-friendly stats. ' +
+				'Get updates on site traffic, successful posts, site searches, and comments, all in real time.'
+		);
+		const connectUrl = addQueryArgs(
+			{
+				url: siteUrl,
+				// TODO: add a parameter to the JPC to redirect back to this step after completion
+				// and in the redirect URL include the ?action=activate_stats parameter
+				// to actually trigger the stats activation action after getting back to JPO
+			},
+			'/jetpack/connect'
+		);
+		const href = ! isConnected ? connectUrl : null;
+
+		return (
+			<Fragment>
+				<FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />
+
+				<TileGrid>
+					<Tile
+						buttonLabel={ translate( 'Activate stats' ) }
+						image={ '/calypso/images/illustrations/type-business.svg' }
+						onClick={ this.handleActivateStats }
+						href={ href }
+					/>
+				</TileGrid>
+			</Fragment>
+		);
+	}
+
+	render() {
+		const { activatedStats, basePath, translate } = this.props;
+
+		return (
+			<div className="steps__main">
+				<DocumentHead title={ translate( 'Jetpack Stats ‹ Jetpack Start' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **13** times:
`translate( 'Homepage ‹ Jetpack Start' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jpo-stats-step)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).